make_android.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
```

Comment:
@luongvo What do you think about using kscript? Under the hood, this templates is more related to kotlin

Reply:
@suho This is the first step of reusing the script from android-templates to build the Android app module only. The next step could be building the main script with Kscript by this task https://github.com/nimblehq/kmm-templates/issues/30.